screens/new-message-screen/index.js

```diff
@@ -100,7 +102,7 @@ const NewMessageScreen = ({ actions, currentUser, navigation, selectedTeamId }:
                                 <Text style={ styles.label }>{ "To:" }</Text>
                                 <Picker
                                     selectedValue={ currentTeamId }
-                                    itemStyle={ { backgroundColor: "#FFFFFF99", color: "black" } }
+                                    itemStyle={ { backgroundColor: "#FFFF99", color: "black" } }
```

Comment:
No clue what original color this wanted to be as the FFFFFF99 original isn't valid.